src/Testing/TestableLivewire.php

```diff
@@ -247,6 +247,7 @@ public function __call($method, $params)
             return $this->macroCall($method, $params);
         }
 
-        return $this->lastResponse->$method(...$params);
+        $this->lastResponse->$method(...$params);
```

Comment:
Can you add a line break after this and an accompanying test?

Reply:
Done!